src/dmd/frontend.d

```diff
@@ -27,24 +27,11 @@ void initDMD()
     import dmd.expression : Expression;
     import dmd.objc : Objc;
     import dmd.builtin : builtin_init;
+    import dmd.compiler : addDefaultVersionIdentifiers, printPredefinedVersions;
 
     global._init;
-
-    version(linux)
-        global.params.isLinux = 1;
-    else version(OSX)
-        global.params.isOSX = 1;
-    else version(FreeBSD)
-        global.params.isFreeBSD = 1;
-    else version(Windows)
-        global.params.isWindows = 1;
-    else version(Solaris)
-        global.params.isSolaris = 1;
-    else version(OpenBSD)
-        global.params.isOpenBSD = 1;
-    else
-        static assert(0, "OS not supported yet.");
-
+    addDefaultVersionIdentifiers();
+    printPredefinedVersions();
```

Comment:
Are we sure that we want to call `printPredefinedVersions` here at all?
After all, this is the `frontend` for DMD as a _library_ code.